admin_guide/backup_restore.adoc

```diff
@@ -177,6 +176,276 @@ that {product-title} was previously installed.
+# cat /usr/lib/systemd/system/etcd.service  | grep ExecStart
+
+ExecStart=/bin/bash -c "GOMAXPROCS=$(nproc) /usr/bin/etcd --force-new-cluster"
+----
++
+Then restart the *etcd* service:
++
+----
+# systemctl daemon-reload
+# systemctl start etcd
+----
+
+[[cluster-restore-multiple-member-etcd-clusters]]
+== Cluster Restore for Multiple-member etcd Clusters
+
+When using an external etcd host, you must first restore the etcd backup
+by creating a new, single node etcd cluster. If using external etcd with
+multiple members, you must then also add any additional etcd members to the
+cluster one by one.
+
+However, the details of the restoration process differ between
+xref:restoring-embedded-etcd[embedded] and
+xref:restoring-external-etcd[external] etcd. See the following
+section and follow the relevant steps
+before
+xref:bringing-openshift-services-back-online[Bringing OpenShift
+Services Back Online].
+
+[[restoring-embedded-etcd]]
+=== Embedded etcd
+
+Restore your etcd backup and configuration:
+
+. Run the following on the master with the embedded etcd:
++
+----
+# ETCD_DIR=/var/lib/origin/openshift.local.etcd
+# mv $ETCD_DIR /var/lib/etcd.orig
+# cp -Rp /var/lib/origin/etcd-backup-<timestamp>/ $ETCD_DIR
+# chcon -R --reference /var/lib/etcd.orig/ $ETCD_DIR
+# chown -R etcd:etcd $ETCD_DIR
+----
++
+[WARNING]
+====
+The `$ETCD_DIR` location differs between external and embedded etcd.
+====
+
+. Create the new, single node etcd cluster:
++
+----
+# etcd -data-dir=/var/lib/origin/openshift.local.etcd \
+    -force-new-cluster
+----
++
+Verify etcd has started successfully by checking the output from the above
+command, which should look similar to the following near the end:
++
+----
+[...]
+2016-06-24 12:14:45.644073 I | etcdserver: starting server... [version: 2.2.5, cluster version: 2.2]
+[...]
+2016-06-24 12:14:46.834394 I | etcdserver: published {Name:default ClientURLs:[http://localhost:2379 http://localhost:4001]} to cluster 5580663a6e0002
+----
+
+. Shut down the process by running the following from a separate terminal:
++
+----
+# pkill etcd
+----
+
+. Continue to xref:bringing-openshift-services-back-online[Bringing
+{product-title} Services Back Online].
+
+[[restoring-external-etcd]]
+=== External etcd
+
+Choose a system to be the initial etcd member, and restore its etcd backup and
+configuration:
+
+. Run the following on the etcd host:
++
+----
+# ETCD_DIR=/var/lib/etcd/
+# mv $ETCD_DIR /var/lib/etcd.orig
+# cp -Rp /var/lib/origin/etcd-backup-<timestamp>/ $ETCD_DIR
+# chcon -R --reference /var/lib/etcd.orig/ $ETCD_DIR
+# chown -R etcd:etcd $ETCD_DIR
+----
++
+[WARNING]
+====
+The `$ETCD_DIR` location differs between external and embedded etcd.
+====
+
+. Restore your *_/etc/etcd/etcd.conf_* file from backup or *_.rpmsave_*.
+
+. Create the new single node cluster using etcd's `--force-new-cluster`
+option. You can do this with a long complex command using the values from
```

Comment:
I'd suggest the "long complex" bit be taken out. Adverbs!

"Do this by using the values from..."